index.js

```diff
@@ -5,13 +5,71 @@ class GATrack {
   static debug = false;
   static trackerName = '';
   static force = null;
+  static V4 = false;
+
+  static async sendEventV4(event_name, event_params) {
```

Comment:
📝 I'd just move this stuff into the normal `sendEvent`. I don't see a point in making V4 it's own method - just seems like it would create duplicate work when we could just call sendEvent and let sendEvent deal with formatting data and sending to both GA endpoints during migration.